packages/kbn-pm/src/commands/destroy.ts

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * and the Server Side Public License, v 1; you may not use this file except in
+ * compliance with, at your election, the Elastic License or the Server Side
+ * Public License, v 1.
+ */
+
+import dedent from 'dedent';
+import del from 'del';
+import ora from 'ora';
+import { join, relative } from 'path';
+
+import { runBazel } from '../utils/bazel';
+import { isDirectory } from '../utils/fs';
+import { log } from '../utils/log';
+import { ICommand } from './';
+
+export const DestroyCommand: ICommand = {
+  description:
+    'Remove the node_modules, remove target directories from all projects, remove extra patterns and runs a Bazel state hard clean.',
+  name: 'destroy',
```

Comment:
The more I think of it, the more I think `reset` is more appropriate. Thoughts?

Reply:
I like it, going to change it!

Reply:
Done 👍